src/app/auth/login/page.tsx

```diff
@@ -0,0 +1,88 @@
+'use client';
+
+import { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import TextInput from '@/components/TextInput/index';
+import { H1, H4Centered, P, AColored } from '@/styles/text';
+import supabase from '@/api/supabase/createClient';
+import COLORS from '@/styles/colors';
+import { FormDiv, QuestionsDiv, SixteenDiv } from '../styles';
+
+export default function Login() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  // const [errorMessage, setErrorMessage] = useState('');
+  const { push } = useRouter();
+  // commenting out Sign Up to use in the next PR -> create a separate sign in page
+  /*
+  const handleSignUp = async () => {
+    const { error } = await supabase.auth.signUp({
+      email,
+      password,
+    });
+
+    if (error) {
+      throw new Error(`An error occurred trying to sign up: ${error.message}`);
+    }
+    push('/');
+  };
+  */
+
+  const handleSignIn = async () => {
+    const { error } = await supabase.auth.signInWithPassword({
+      email,
+      password,
+    });
+
+    if (error) {
+      throw new Error(`An error occurred trying to sign in: ${error.message}`);
+    }
+    push('/');
+  };
+
+  return (
+    <FormDiv>
```

Comment:
<img width="726" alt="Screenshot 2023-10-28 at 9 52 26 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/47801900/ec2876e7-c9e4-4222-a29f-21fd94f8e78d">

We may want to handle this in a future PR (possibly the `layout.tsx` one) but wondering if it would be possible to resize and center the blue border div so the screen looks like the one in the design?

When I initially wrote this PR I only added the header/questions/button on the Linear issue, with the assumption that the blue div & positioning would be handled in a future sprint. However since we already have the blue div here (and it looks great!) perhaps we can just work on positioning right now.

Reply:
sounds good! I didn't know if that was supposed to be handled in the layout, or in the div. Let me actually do my reading on layouts 😅 to see if my statement makes any sense. but otherwise, yes, i'll def include the div positioning in this sprint.

Reply:
i think a standardized div to accomplish the blue box should be okay and we can avoid using a layout file. usually we can keep the layout for a consistent component that we'll see across multiple pages (ex. navbars, headers, etc.)

Reply:
also in a similar vein for this task, this might involve starting to use media queries but not sure if this is exactly necessary and i also think that building out responsiveness can be a later task. @pragyakallanagoudar thoughts?